### ISSUE TITLE
Tensorflow readme error

### ISSUE THREAD

**Issue by ujsyehao** [Microsoft/MMdnn#301]
The usage `mmtoir -f tensorflow -w mobilenet_v1_1.0_224/frozen_graph.pb --inNodeName input --inputShape 224,224,3 --dstNodeName MobilenetV1/Predictions/Softmax -o mobilenet_v1`
will cause error:
![173](https://user-images.githubusercontent.com/17454232/42501476-6789c6b4-8466-11e8-9991-dfe83ac36e85.png)
![174](https://user-images.githubusercontent.com/17454232/42501491-6d726ffe-8466-11e8-88f6-57f8dbb68978.png)
**It shoule be `--frozen_pb *.pb` and `224 224 3`.**

**Comment by namizzz** [comment]
Hi @ujsyehao ,please try the newest code of MMdnn:
```cmd
pip install -U git+https://github.com/Microsoft/MMdnn.git@master
```
README is for newest code.

**Comment by ujsyehao** [comment]
@namizzz Okay, thanks!
